client/src/router/index.ts

```diff
@@ -0,0 +1,21 @@
+import Vue from 'vue'
+import VueRouter from 'vue-router'
+import Landing from '../views/landing.vue'
+
+Vue.use(VueRouter)
+
+const routes = [
```

Comment:
You might want to extract routes into a separate file so you can use it during unit testing.